### ISSUE TITLE
Gems do not load since split

### ISSUE THREAD

**Issue by dylan-azucena** [Azure/azure-storage-ruby#111]
Following the instructions on the README, the new gems will not load via `require`.

After `gem install azure-storage-blob`, doing `require "azure/storage/blob"` leads to:

```
/Users/dazucena/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- azure/storage/blob (LoadError)
	from /Users/dazucena/.rbenv/versions/2.4.1/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from test_azure_api.rb:2:in `<main>'
```

The same is true for azure-storage-table, etc.
I have verified that `azure-storage-table` appears in `gem list`.

**Comment by katmsft** [comment]
Hi, the issue is found. There are some problems with the current release process and making the gem wrongly.
The issue will be fixed soon. Thank you very much for reporting.

**Comment by katmsft** [comment]
We are currently trying to push the fix but struggling because [an Travis-CI issue](https://github.com/travis-ci/travis-ci/issues/9088) preventing us from merge the PR. Please stay tuned for the fix.

**Comment by dylan-azucena** [comment]
Understood, thank you for the quick response!

**Comment by katmsft** [comment]
Hi @dylan-azucena , again thanks for reporting this issue. The issue is fixed in 1.0.1 for all packages. Feel free to upgrade your gems and the problems should be resolved.

**Comment by dylan-azucena** [comment]
@katmsft upgraded gems and was able require properly. Thank you!
